app/models/server_group.rb

```diff
@@ -0,0 +1,4 @@
+class ServerGroup < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  has_many :vms
```

Comment:
Probably want to have a `:dependent => :nullify` so we don't leave hanging references